managers/xbps.go

```diff
@@ -0,0 +1,36 @@
+package managers
+
+// NewXbps creates a new Manager instance.
+func NewXbps() *Manager {
+	return &Manager{
+		commands: ManagerCommands{
+			clean:   "xbps-remove",
+			install: "xbps-install",
+			refresh: "xbps-install",
+			remove:  "xbps-remove",
+			update:  "sh",
+		},
+		flags: ManagerFlags{
+			global: []string{},
+			clean: []string{
+				"--yes",
+				"--clean-cache",
+			},
+			install: []string{
+				"--yes",
+			},
+			refresh: []string{
+				"--sync",
+			},
+			remove: []string{
+				"--yes",
+				"--recursive",
+				"--remove-orphans",
+			},
+			update: []string{
+				"-c",
+				"xbps-install --yes --update && xbps-install --yes --update",
```

Comment:
Before someone asks: This needs to be called twice because if xbps detects that there's an update for itself, it will update *only itself* (and dependencies) and terminate. I was unable to find an option that forces xbps to both update itself *and* the rest of the system in one go.